JBBCode/CodeDefinition.php

```diff
@@ -11,28 +11,28 @@
  */
 class CodeDefinition
 {
-    /* NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
+    /* @var string NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
```

Comment:
For reference: https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#55-examples